docs/self-managed/concepts/multi-region/dual-region.md

```diff
@@ -9,141 +9,125 @@ description: "A dual-region setup allows you to run Camunda in two regions synch
+| Camunda Platform Configuration | <p>The overall Camunda platform is **active-passive**</p><p><ul><li>**Active-Passive Traffic Handling:** One active and one passive region serve active user traffic.</li><li>**Traffic to Both Regions:** Serving traffic to both regions will cause component detachment, potentially resulting in different data visibility in Operate and Tasklist.</li></ul></p>                                                                                                                   |
+| Identity Support               | Identity, including multi-tenancy and Role-Based Access Control (RBAC), is currently unavailable in this setup.                                                                                                                                                                                                                                                                                                                                                                         |
 | Optimize Support               | Not supported (requires Identity).                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
-| Connectors Deployment          | Connectors can be deployed in a dual-region setup, but attention to [idempotency](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event) is required to avoid event duplication. In a dual-region setup, you'll have two connector deployments and using message idempotency is of importance to not duplicate events.                                                                                                                                  |
+| Connectors Deployment          | Connectors can be deployed in a Dual-Region setup, but attention to [idempotency](../../../components/connectors/use-connectors/inbound.md#creating-the-connector-event) is required to avoid event duplication. In a dual-region setup, you'll have two connector deployments and using message idempotency is of importance to not duplicate events.                                                                                                                                  |
```

Comment:
maybe for the technical writers to have a look over.

I would have kept it at `dual-region` or maximum at `Dual-region` (looks odd to me mid sentence) but at least `Dual-Region` I was shunned for in the past.